internal/support/notifications/v2/controller/http/subscription_test.go

```diff
@@ -93,6 +94,32 @@ func addSubscriptionRequestData() requests.AddSubscriptionRequest {
 	return testAddSubscriptionReq
 }
 
+func updateSubscriptionRequestData() requests.UpdateSubscriptionRequest {
+	testUUID := ExampleUUID
+	testName := testSubscriptionName
+	testDescription := testSubscriptionDescription
+	var testUpdateSubscriptionReq = requests.UpdateSubscriptionRequest{
```

Comment:
Use new factory method

Reply:
Updated subscription_test to use new factory method.